source/src/ports/generic_networkhandler.c

```diff
@@ -240,7 +240,7 @@ EipStatus NetworkHandlerInitialize(void) {
 
   /* have QoS DSCP explicit appear on UDP responses to broadcast messages */
   if (SetQosOnSocket( g_network_status.udp_global_broadcast_listener,
-                     GetPriorityForSocket(0xFFFF) ) != 0) {
+                     CipQosGetDscpPriority(0xFFFF) ) != 0) {
```

Comment:
replace magic number

Reply:
as already commented before the code lacks kConnectionObjectPriorityExplicit

Reply:
Thanks, this is part of the "replace magic number" reminder :)